packages/modular-scripts/src/__tests__/esmView.test.ts

```diff
@@ -737,7 +799,83 @@ describe('modular-scripts', () => {
             path.join(modularRoot, 'dist', 'sample-esm-view', 'package.json'),
           )) as CoreProperties
         ).module,
-      ).toEqual('./static/js/index-5PFA727O.js');
+      ).toEqual('./static/js/index-CWZPL7BC.js');
+    });
+  });
+
+  describe('WHEN we specify an allow / block list', () => {
```

Comment:
This test file is very long, and these unit tests may get lost. I think it would be better to put them in a file that makes it clear this is testing `filterDependencies.ts`.

Reply:
That's fair enough, though I would have been happy if you'd created a to-be-completed test file as we can always flesh it out later!